CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+- update `firebase init functions` flow to be codebase-aware.
```

Comment:
Please capitalize first sentence! Also, I think the rule is one changelog entry per PR. Since something like adding unit test isn't that relevant to the end-user of the CLI, I'd remove the entry.